recipes/faqcs/meta.yaml

```diff
@@ -17,7 +17,7 @@ requirements:
     - {{ compiler('cxx') }}
   host:
     - zlib
-    - r-base
+    - r-base >=3.3
```

Comment:
This is actually implicit, packages are only built for a single R version range (at the moment 3.6.x).

Reply:
OK. I should revert it back